packages/app/i18n/en.pot

```diff
@@ -34,6 +34,9 @@ msgstr ""
 msgid "Update"
 msgstr ""
 
+msgid "Add to"
```

Comment:
Looks like this needs to be re-generated again, running `yarn build` should do it.

Reply:
Yeah sorry for missing the regeneration again, I can't get it to do it automatically, seems `yarn build` doesn't do it for me. Only way I've found so far is to delete it before the build.